project/frontend/src/index.js

```diff
@@ -24,7 +23,6 @@ ReactDOM.render(
           <Route exact path='/' component={HomePage} />
           <Route path='/find-match' component={PreferenceForm} />
           <Route path='/match-results' component={MatchResults} />
-          <Route path='/mapRoute' component={MapContainer} />
```

Comment:
Make this hyphenated like the other paths rather than camel case I think

Reply:
this line was removed in this PR so I don't think there's a point in changing it now!

Reply:
oh oops you're right